syntax = "proto3";
package ServerPacket;

message ClientInfo {
  int32 Socket = 1;
  string NickName = 2;
}